pkg/network/ovn_kubernetes.go

```diff
@@ -231,6 +231,11 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 		conf.DefaultNetwork.OVNKubernetesConfig = &operv1.OVNKubernetesConfig{}
 	}
 
+	var encapOverhead uint32 = 100 // 100 byte geneve header
```

Comment:
nit: use a constant?  If possible, let the compiler calculate this value based on all the components that make it up.

Reply:
Good idea.

Reply:
I did this to some extent. I need to split up the PRs see - https://github.com/openshift/cluster-network-operator/pull/928